src/main/java/core/basesyntax/FigureDrawer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDrawer {
```

Comment:
```suggestion
public interface Drawer {
```

As we can use this interface for other classes, not only Figures